arches/app/datatypes/datatypes.py

```diff
@@ -2235,6 +2235,20 @@ def validate(self, value, row_number=None, source="", node=None, nodeid=None, st
                     errors.append({"type": error_type, "message": message})
         return errors
 
+    def pre_tile_save(self, tile, nodeid, **kwargs):
+        node = kwargs.pop("node", None)
```

Comment:
I'm pretty sure that we shouldn't need lines 2239-2245 as no other type of node, besides "resource-instance", and "resource-instance-list",  should trigger this method.

If that is the case, I'm not sure if adding the **kwargs param is necessary any more.